cvat/apps/engine/static/engine/js/shapes.js

```diff
@@ -3508,6 +3527,10 @@ class RaysView extends PolyShapeView {
         this._uis.shape = this._scenes.svg.group()
             .fill(this._appearance.fill || this._appearance.colors.shape)
             .stroke(this._appearance.stroke || this._appearance.colors.shape)
+            .attr({
+                'stroke-width': STROKE_WIDTH / window.cvat.player.geometry.scale,
```

Comment:
limit maximum width to something sensible

Reply:
I think it's limited since `STROKE_WIDTH` is a constant and `window.cvat.player.geometry.scale` is within zoom boundaries.
This line follows the pattern in CVAT of making line width scale properly on zoom.